ext/page_log/palite/CMakeLists.txt

```diff
@@ -43,6 +43,13 @@ target_compile_options(
     PRIVATE
         ${COMPILER_DIAGNOSTIC_CXX_FLAGS}
 )
+if(NOT MSVC)
```

Comment:
I would recommend using WT variables for compiler tests. In this case it would be:
```cmake
if (NOT MSVC_C_COMPILER)
    ...
```